admin/app/views/commercial/commercialMenu.scala.html

```diff
@@ -111,7 +111,7 @@
                     <ul>
                         <li><a href="https://github.com/guardian/commercial-tools">Commercial tools repo</a></li>
                         <li><a href="https://dfp-playground.appspot.com/59739193">DFP API playground</a></li>
-                        <li><a href="http://kibana.gu-web.net/goto/1955d282a7b6cc452b673cdb9c27f1af">
+                        <li><a href="https://kibana.gu-web.net/goto/1955d282a7b6cc452b673cdb9c27f1af">
```

Comment:
There is a property for the kibana url so you don't have to hardcode it. 💃 https://github.com/guardian/frontend/blob/master/common/app/common/configuration.scala#L544